.github/workflows/docker_update_theia.yml

```diff
@@ -29,18 +32,24 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Push antrea/theia-clickhouse-operator
+      - name: Push antrea/clickhouse-operator
         uses: akhilerm/tag-push-action@v2.1.0
         with:
           src: docker.io/altinity/clickhouse-operator:${{ github.event.inputs.ch-operator-tag }}
           dst: |
-            docker.io/antrea/theia-clickhouse-operator:${{ github.event.inputs.ch-operator-tag }}
-      - name: Push antrea/theia-metrics-exporter
+            docker.io/antrea/clickhouse-operator:${{ github.event.inputs.ch-operator-tag }}
+      - name: Push antrea/metrics-exporter
         uses: akhilerm/tag-push-action@v2.1.0
         with:
           src: docker.io/altinity/metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
           dst: |
-            docker.io/antrea/theia-metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
+            docker.io/antrea/metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
+      - name: Push antrea/clickhouse-server
```

Comment:
Maybe add a comment saying we need this to save official image to decouple the Antrea e2e test dependency on Thiea repo?

Reply:
done